schemas/subscription-results.v1.schema.json

```diff
@@ -0,0 +1,125 @@
+{
+   "$schema":"http://json-schema.org/draft-07/schema#",
+   "$ref":"#/definitions/Main",
+   "definitions":{
+      "Main":{
+         "title":"subscription_results",
```

Comment:
@untitaker This type doesn't seem to be getting generated correctly

Reply:
what is wrong about it? I'm looking at `SubscriptionResults` in the resulting schema, and it looks like this:

```

class SubscriptionResults(TypedDict, total=False):
    """subscription_results."""

    version: Required[int]
    """required"""

    payload: Required[Union["PayloadV3", "PayloadV2"]]
    """required"""


```